### PR TITLE
Add more detailed comment about _assign_attribute method [ci skip]

### PR DESCRIPTION
**Pull request by meinac**

### CODE REVIEW

File: activerecord/lib/active_record/attribute_assignment.rb [hunk 1]
```diff
@@ -29,7 +29,8 @@ def _assign_attributes(attributes) # :nodoc:
       assign_multiparameter_attributes(multi_parameter_attributes) unless multi_parameter_attributes.empty?
     end
 
-    # Re-raise with the ActiveRecord constant in case of an error
+    # Tries to assign given value to given attribute
```
Comment (rafaelfranca) [inline activerecord/lib/active_record/attribute_assignment.rb]: Missing `.` in the end of the line

File: activerecord/lib/active_record/attribute_assignment.rb [hunk 2]
```diff
@@ -29,7 +29,8 @@ def _assign_attributes(attributes) # :nodoc:
...
+    # If encounters with an error re-raises with the ActiveRecord constant
```
Comment (rafaelfranca) [inline activerecord/lib/active_record/attribute_assignment.rb]: `In case of an error, re-raises with the ActiveRecord constant.`

**Comment by meinac** [comment]
@rafaelfranca you were always helpful for my pull requests to find the best way, thank you. I've fixed problems about this pr.

**Comment by rafaelfranca** [comment]
You are welcome. Thank YOU for improving Rails :heart:

**Pull request merged**